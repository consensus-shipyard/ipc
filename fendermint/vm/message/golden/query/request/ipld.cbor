a16449706c64d82a582300122023d404cce8344faaf4c618fc73867998946c9b074ca10d5d37b654a186d67d4b