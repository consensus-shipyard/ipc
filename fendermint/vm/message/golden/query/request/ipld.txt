Ipld(Cid(QmQkWVLf1rDgKJKd3G3G4v5V92q8JhoSTDQR2QDDgXUaKU))